src/Nox.ClientApi.Tests/Generated/Nox.Generator/Nox.Generator.NoxCodeGenerator/Application.IntegrationEvents.CountryCreated.g.cs

```diff
@@ -12,4 +13,5 @@ namespace ClientApi.Application.IntegrationEvents;
 /// <summary>
 /// CountryCreated integration event.
 /// </summary>
+[IntegrationEventType("Created", nameof(Country))]
```

Comment:
For the Default IE, let's make the name lowercase, for Custom IntegrationEvents let's make it Pascal Case (Lower case first letter only)
```suggestion
[IntegrationEventType("created", nameof(Country))]
```